File: src/main/java/com/example/mapstruct/demo/entity/Task.java
package com.example.mapstruct.demo.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Entity
@Table(name = "task")
@Data
public class Task {
    @Id
    private Long id;
    private String description;
}

File: src/main/kotlin/com/example/mapstruct/demo/entity/Task.kt
package com.example.mapstruct.demo.entity

import jakarta.persistence.*

@Entity
@Table(name = "task")
class Task {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private val id: Long? = null
    private val description: String? = null

    override fun toString() = "(id = $id, description = $description)"
}
